tests/suites/verify.js

```diff
@@ -101,13 +101,23 @@ export function verifySuite({
             disclosed?.invalid?.proofTypeAndCryptosuite);
           await verificationFail({credential, verifier});
         });
+        it('If proofConfig.created is set and if the value is not a valid ' +
```

Comment:
```suggestion
        it('If proofConfig.created is set and the value is not a valid ' +
```

Reply:
@TallTed we try to match the spec as exactly as possible, so I hope you don't mind, but I made a PR here: https://github.com/w3c/vc-di-bbs/pull/175/files

Reply:
I think this should not be marked resolved until https://github.com/w3c/vc-di-bbs/issues/178 is resolved.

Reply:
@TallTed as this PR is part of a chain with other tests in it perhaps we should merge this as is and fix once that issue has been addressed and the group can agree on consensus on spec text? The big issue I'm running into here is that there are 4 more PRs branched from this branch which build on the features uses in this PR, and some of those PRs contain tests and features that would improve main and make the suite extensible. So it seems like we need to monitor and update test titles more.

Reply:
OK; merge this one. Agreement between test and spec language is now being tracked in https://github.com/w3c/vc-di-bbs-test-suite/issues/48